package.json

```diff
@@ -38,42 +38,43 @@
     "deep-rename-keys": "^0.1.0",
     "del": "^2.2.0",
     "dot": "^1.0.3",
-    "dotenv": "^2.0.0",
-    "epochtalk-core-pg": "^1.0.8",
```

Comment:
this merge was resolved incorrectly.

you should take care in doing this next time.  it was confusing to see this change happen in the previous commit, and then again in the next commit.